packages/expo-constants/src/Constants.ts

```diff
@@ -1,17 +1,25 @@
 import { ExpoConfig } from '@expo/config-types';
+// @ts-ignore -- optional interface, will gracefully degrade to `any` if not installed
+import type { Manifest as DevLauncherManifest } from 'expo-dev-launcher';
+import type {
+  BareManifest,
+  EASConfig,
+  ExpoGoConfig,
+  LegacyManifest,
+  NewManifest,
+  // @ts-ignore -- optional interface, will gracefully degrade to `any` if not installed
+} from 'expo-manifests';
```

Comment:
nope! since this uses `import type` it is considered an optional dependency and will gracefully degrade to `any` if not installed. (see note in summary)